find_me_a_sandwich/app/views/favorites/show.html.haml

```diff
@@ -0,0 +1,18 @@
+.page-header
+  %h1 Favorites
+
+- if user_signed_in?
+  - if @favorites.none?
+    %p Save some meals to favorites and they will show up here after.
+  - else
+    %table{ class: "table" }
+      %thead
+        %tr
+          %th{ colspan: "2" } Name
```

Comment:
This is not an issue, but there is no need for the quotes here.  Haml is pretty flexible.